browser/farbling/brave_navigator_keyboard_api_browsertest.cc

```diff
@@ -0,0 +1,134 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "base/containers/contains.h"
+#include "base/path_service.h"
+#include "brave/browser/brave_content_browser_client.h"
+#include "brave/common/brave_paths.h"
+#include "brave/components/brave_shields/browser/brave_shields_util.h"
+#include "chrome/browser/content_settings/host_content_settings_map_factory.h"
+#include "chrome/browser/profiles/profile.h"
+#include "chrome/browser/ui/browser.h"
+#include "chrome/common/chrome_content_client.h"
+#include "chrome/test/base/in_process_browser_test.h"
+#include "chrome/test/base/ui_test_utils.h"
+#include "components/network_session_configurator/common/network_switches.h"
+#include "components/permissions/permission_request.h"
+#include "content/public/test/browser_test.h"
+#include "content/public/test/browser_test_utils.h"
+#include "net/dns/mock_host_resolver.h"
+#include "url/gurl.h"
+
+using brave_shields::ControlType;
+
+const char kGetLayoutMapScript[] =
```

Comment:
could be constexpr and also raw string literarls are handier

Reply:
Changed in 0dfc2a865d79406aacbb4dbf353e0b91f6d3846b. Also changed the script itself as it was failing on Windows CI with 
<pre>
15:11:59  ../../brave/browser/farbling/brave_navigator_keyboard_api_browsertest.cc(116): error: Expected equality of these values:
15:11:59  
15:11:59    "a"
15:11:59  
15:11:59      Which is: 00007FF7741DC7BD
15:11:59  
15:11:59    EvalJs(contents(), kGetLayoutMapScript)
15:11:59  
15:11:59      Which is: null
</pre>
Assuming, because there's no physical keyboard?